.github/workflows/tests.yml

```diff
@@ -55,10 +56,12 @@ jobs:
             composer-
 
       - name: Download dependencies
-        run: |
-          composer update --no-interaction --no-progress --optimize-autoloader
-          ./vendor/bin/simple-phpunit install
+        run: composer update --no-interaction --no-progress --optimize-autoloader
 
-      - name: Run tests with PHPUnit
-        run: |
-          ./vendor/bin/simple-phpunit
+      - name: Run tests
+        if: ${{ '8.1' != matrix.php-version }}
+        run: ./vendor/bin/simple-phpunit
+
+      - name: Run tests for php 8.1
+        if: ${{ '8.1' == matrix.php-version }}
+        run: SYMFONY_PHPUNIT_VERSION=9.5.5 ./vendor/bin/simple-phpunit
```

Comment:
phpunit fixed the following deprecation in version [`9.5.5`](https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-9.5.md). 
```
  1x: PHPUnit\Runner\DefaultTestResultCache implements the Serializable interface, which is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary)
    1x in ClassLoader::loadClass from Composer\Autoload
```
If I would not use this specific phpunit version for php8.1, the execution of the tests would fail.

Reply:
Better to have 9.5 as the version here. It'll get the latest 9.5.x.